tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserIntegerTests.cs

```diff
@@ -685,15 +685,10 @@ public unsafe void ParseUInt64Hex(string text, bool expectSuccess, UInt64 expect
         [InlineData("-0128", true, -128, 5)] // min
         [InlineData("0128", false, 0, 0)] // positive overflow test
         [InlineData("-0129", false, 0, 0)] // negative overflow test
-        [InlineData("111", true, 111, 3)]
```

Comment:
Why are these changes here?

Reply:
Yes, these test cases are duplicate. It was observed when VS test discovery stated explicitly that the tests have already been added to the test window and hence ignoring it.

Line 688 is: `[InlineData("111", true, 111, 3)]`
Which is the same as line 649 - https://github.com/dotnet/corefxlab/pull/1725/files/d6f7d417dcd26ee92a6f13795ee58273fe24bf26#diff-de2a92cba24dadec8a492537ca278d1bL649